### PR TITLE
[FEAT] : Prisma Integration

### PR DESCRIPTION
**Pull request by BharathSanjeeviT**
## Description
Integrating Prisma ORM into our project will bring numerous benefits, including faster and more reliable queries, flexible data modeling, and easier management of the database. This will allow developers to focus more on building features rather than dealing with complex queries, making our project more scalable and user-friendly.

## Changes Made
I have spent significant time testing and integrating Prisma into our codebase, ensuring compatibility and adherence to established code conventions.

## Impact
The integration of Prisma will enhance the performance and functionality of our project, and I am confident that it will help us to better serve our users. 

## Testing Done
I have tested the integration thoroughly and made sure it works seamlessly with our existing codebase.

## Request for Review
I would appreciate your feedback on this Pull Request.

## Screenshots / Videos

https://user-images.githubusercontent.com/119755499/228765402-2c1b0235-ac16-4e3e-9587-6c62bff238c2.mp4

### CODE REVIEW

**Comment by BharathSanjeeviT** [comment]
Hey @RahulNavneeth and @navi-prem , Please take a look at this PR, Thanks 😄

File: pages/api/index.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { prisma } from "@/libs/prisma";
+const handler = async (req,res) => {
+    if(req.method==='POST'){
+        console.log(req.body.toShort);
+        const ra = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890"
```
Comment (RahulNavneeth) [inline pages/api/index.ts]: Use more viable variable name instead to `ra`

**Review by RahulNavneeth — COMMENTED** [review]
PTAL

File: pages/api/index.ts (hunk 1 above)
Comment (BharathSanjeeviT) [reply pages/api/index.ts]: Hey @RahulNavneeth , 
    Made the concurrent changes you proposed 😄

**Comment by RahulNavneeth** [comment]
@BharathSanjeeviT For future PR please add validation for existing url 😄

**Comment by RahulNavneeth** [comment]
LGTM 🌟

**Pull request merged**